baseplate/secrets/fetcher.py

```diff
@@ -18,8 +20,15 @@
         secret/three,
 
 where each secret is a path to look up in Vault. The daemon authenticates with
-Vault as the EC2 server it is running on. Vault can map that context to
-appropriate policies accordingly.
+Vault as a role using a token obtained from an auth backend designated by `auth_type`.
+
+Currently supported auth types:
+    - aws: uses an AWS-signed instance identity document from the instance
```

Comment:
could you build the docs locally (`make docs` in the VM then go to `http://baseplate.local`) and make sure this list renders ok? reStructuredText can be a bit finnicky